Charcoal/Charcoal/Library.cpp

```diff
@@ -87,6 +87,7 @@ std::string Library::add(PWSTR path)
     if (f == "epub")
     {
         Epub e;
+        e.grayscaleEpub(path);
```

Comment:
This line should be removed

Reply:
I added an additional commit to resolve issue.